### DISCUSSION TITLE
I AM LOOKING FOR A GROUP OF PEOPLE TO PROGRAM TOGETHER AND HELP EACH OTHER

### DISCUSSION THREAD

**Discussion by Karounos** [community/community#28215] [Other Feature Feedback, Questions, & Ideas]
If anybody knows a place where I can find programmer peers I would appreciated...

I am a beginner coder working with C and I  am a teen

**Comment by atuschek0** [comment]
Are you in school? It may help to try and start a club or extracurricular in order to find peer that also share this interest. If a club won't work then it may be ask peers you study with if they want to form a study group.

**Comment by Karounos** [comment]
The thing is that my school doesn't care and my friends are not tech guys

**Comment by atuschek0** [comment]
If you mean meeting up with real people this will probably be difficult for you as a teen. Are there any programming courses or bootcamps in your area you are able to attend? Usually college/university years is when it's much easier to form groups with people that share like-minded goals/concentrations. Also, see if theres any tech conferences happening in your area.

You're on GitHub which is good, you are able to contribute to open-source with people all around the world. Try and find an open source group that is open and friendly, read their code, try to understand it, and learn! If the open-source group you find is friendly, contribute to their discussions if you have any questions about anything.

Congratulations on starting early :)
